proto/build_event_stream.proto

```diff
@@ -1456,6 +1456,7 @@ message WorkflowConfigured {
 // Event describing a workflow command that completed.
 // Note: the event ID holds the invocation ID that identifies the command.
 // Next Tag: 6
+// TODO(Maggie): Deprecate - duplicate with ChildInvocationCompleted
```

Comment:
Good idea - I also cleaned up the FE reference to this event (it's unneeded because ChildInvocationCompleted currently contains the same data) and added a TODO to rename it to `RemoteRunnerStepCompleted`